packages/shared/src/components/squads/PublicSquadSubmissionActions.tsx

```diff
@@ -82,7 +82,9 @@ const PublicSquadSubmissionActions = (
           onClick={onSubmit}
           type="button"
         >
-          Submit for review
+          {status === SquadStatus.Approved
```

Comment:
We should not have this step anymore. Anything related to public squad request should be removed.

Reply:
I am planning to raise a separate PR pointed to this to keep it approved-PRs only.